.cppcheck/includes.txt

```diff
@@ -0,0 +1,92 @@
+/concord-bft/
```

Comment:
Can't we calculate the work dirs automatically?
We'll forget to update this when additional directories are added.
Isn't it better to use excludes.txt?

Reply:
You are right, I was trying to do so and wrote a script which created an include file according to all hpp/h files. I removed it later, since it didnt work well - as many includes are relative, e.g #include "categorization/db_categories.h"

The way to know if includes are missing is to run --check-config which runs a special run and give comments on all configuration issues, some of them are includes. Tha'ts what I did.
To have an auto discovering script will require much more work (dig into CMakeLists.txt files). I might try to do it on a 2nd iteration. Anyhow, Cppcheck will never fail if include is not find - it just prints an error.

Reply:
Doesn't this utility support using cmake's compilation database like clang-tidy?
https://cppcheck.sourceforge.io/manual.pdf
see chapter 4.2.

Reply:
That's a good point. For that, we will have to define target  project folder as well (I need to do more research).
I don't want to loose the other options, since they allow checking single files + wildecards and folders, not part of a project, which are important for manual checks.
I've already changed to this option as the main one. There is one thing that compicates thing a bit, project import does not support ignore files, so I have to generate all messages/cmf and protobufs (1-2 minute of build, if you havn't built already).I will see if the includes are needed - i'm not sure they are needed anymore. thanks.